tests/assets/missing-value-date-sanitized.ics

```diff
@@ -20,4 +20,19 @@ DTEND;VALUE=DATE:20161006
 DTSTAMP:20161004T144437Z
 SEQUENCE:0
 END:VEVENT
+BEGIN:VEVENT
+UID:Caldav-7-374
+X-SMT-CATEGORY-COLOR:-16743735
+CATEGORIES:Personal
+LAST-MODIFIED:19700101T000000Z
+TRANSP:OPAQUE
+DTSTART:20210608T183000Z
+DTEND:20210608T210000Z
+X-SMT-MISSING-YEAR:0
+DTSTAMP:20230503T144306Z
+STATUS:CONFIRMED
+RRULE:FREQ=WEEKLY;INTERVAL=1;BYDAY=TU
+EXDATE;VALUE=DATE:20230328
```

Comment:
**issue (blocking):** You need to but the same thing with the bad form (`EXDATE:20230418` in `missing-value-date.ics` for the test to work.

Reply:
I did that locally. No idea why it wasn't committed.

Reply:
Classic: forgot to save the file before committing. Change was still present in my editor.